ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/HiveMetaStoreAuthorizer.java

```diff
@@ -213,15 +214,46 @@ public final List<String> filterCatalogs(List<String> catalogs) throws MetaExcep
 
   @Override
   @Deprecated
-  public List<TableMeta> filterTableMetas(String catName, String dbName,List<TableMeta> tableMetas)
+  public List<TableMeta> filterTableMetas(String catName, String dbName, List<TableMeta> tableMetas)
```

Comment:
In this method, I see that "catName" is never used, so what would be the reason to have this parameter?

Reply:
It is a deprecated method from the interface MetaStoreFilterHook, it has to be implemented. As for the usage of it, I would say removing the parameter would be outside of the scope of this review (looking around the codebase, most people pass null to the method, so I'm going to guess it is not really used anywhere but fixing it should be a different PR).

Reply:
Got it. Thanks